examples/job/expansions/README.md

```diff
@@ -0,0 +1,237 @@
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/job/expansions/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Example: Multiple Job Objects from Template Expansion
+
+In this example, we will run multiple Kubernetes Jobs created from
+a common template.  You may want to be familiar with the basic,
+non-parallel, use of [Job](../../../docs/user-guide/jobs.md) first.
+
+## Basic Template Expansion
+
+First, create a template of a Job object:
+
+<!-- BEGIN MUNGE: EXAMPLE job-template.yaml -->
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Job
+metadata:
+  name: process-item-$ITEM
+spec:
+  selector:
+    matchLabels:
+      app: jobexample
+      item: $ITEM
+  template:
+    metadata:
+      name: jobexample
+      labels:
+        app: jobexample
+        item: $ITEM
+    spec:
+      containers:
+      - name: c
+        image: busybox
+        command: ["sh", "-c", "echo Processing item $ITEM && sleep 5"]
+      restartPolicy: Never
+```
+
+[Download example](job-template.yaml?raw=true)
+<!-- END MUNGE: EXAMPLE job-template.yaml -->
+
+Unlike a *pod template*, a job template is not a Kubernetes API type.  It is just
+a Job object that has some placeholders that need to be filled in before it
+can be used.
+
+In this example, the only processing the container does is to `echo` a string and
+sleep for a bit.
+
+In a real use case, the processing would be some substantial computation, such as
+rendering a frame of a movie, or processing a range of rows in a database.
+The "$ITEM" parameter would specify for example, the frame number or the row range.
+
+This Job has two labels. The first label, with key `app`, distinguishes this group of jobs from
+other groups of jobs (these are not shown, but there might be other ones).  The second key (`item`)
+distinguishes individual jobs in the group.  Neither of these keys are special to kubernetes --
+you can pick your own label scheme.  Just make sure each job has a different label selector, from
+every other job.
+
+Next, expand the template into multiple files, one for each item to be processed.
+
+```console
+# Expand files into a temporary directory
+$ mkdir ./tmp
```

Comment:
The ls, and create are using `jobs` dir, but here you're using tmp, which is also with sed.